tdvt/tdvt/tdvt.py

```diff
@@ -45,43 +46,46 @@ class TestOutputFiles(object):
     combined_output = []
 
     @classmethod
-    def copy_output_file(c, src_name, src_dir):
+    def copy_output_file(cls, src_name, src_dir, csv_dialect):
         src = os.path.join(src_dir, src_name)
         logging.debug("Copying {0} to output".format(src))
         try:
             with open(src, 'r', encoding='utf8') as src_file:
-                reader = csv.DictReader(src_file, dialect='tdvt')
+                reader = csv.DictReader(src_file, dialect=csv_dialect)
```

Comment:
Eventually I think we should consider replacing the term dialect with format within this TDVT usage.

Reply:
That's a very good point. "Dialect" is consistent with Python, but "format" makes more sense. I can do it in this PR.